### PR TITLE
Feature/mvp2 223 show sharable receipt when user places bet

### PR DESCRIPTION
**Pull request by fx1dev**
https://wallfair-product.atlassian.net/browse/MVP2-223
https://wallfair-product.atlassian.net/browse/MVP2-345
https://wallfair-product.atlassian.net/browse/MVP2-346

### CODE REVIEW

File: src/components/BetApproveView/index.js [hunk 1]
```diff
@@ -82,6 +104,31 @@ const BetApproveView = ({ visible, hidePopup }) => {
...
+        <div className={classNames(styles.entry)}>
+          <div className={styles.label}>Amount placed</div>
+          <div className={styles.value}>
+            {amountPlaced} <span>WFAIR</span>
+          </div>
+        </div>
+        <div className={classNames(styles.entry)}>
+          <div className={styles.label}>Potential outcome</div>
+          <div className={styles.value}>
+            {potentialOutcome} <span>WFAIR</span>
```
Comment (haris-bravo-klika) [inline src/components/BetApproveView/index.js:118]: Also, please use the convert helper to convert `amountPlaced`, `potentialOutcome`, and all other values which represent a monetary value in order to show the user the correct amounts.

File: src/components/BetApproveView/index.js [hunk 2]
```diff
@@ -106,9 +168,11 @@ const mapDispatchToProps = dispatch => {
 };
 
 const mapStateToProps = state => {
+  console.log('state', state);
```
Comment (haris-bravo-klika) [inline src/components/BetApproveView/index.js:171]: rm log

File: src/components/BetApproveView/styles.module.scss [hunk 3]
```diff
@@ -11,6 +11,77 @@
     color:            $colorPrimaryDark;
     border-radius:    10px;
 
+  .ShareButtonContainer {
+    padding-top:   10px;
+    padding-bottom:   50px;
+
+    .shareButton {
+      display: flex;
+      //width: 100px;
```
Comment (haris-bravo-klika) [inline src/components/BetApproveView/styles.module.scss:20]: rm commented rule

File: src/components/HomeSettings/index.js [hunk 4]
```diff
@@ -107,6 +111,37 @@ const HomeSettings = ({
...
+            className={styles.goIntoSettingIcon}
+          />
+        </div>
+        <div
+          className={classNames(styles.closeProfileContainer)}
+          onClick={onCloseProfile}
+        >
+          <Icon
+            iconType={IconType.cross}
+            // onClick={}
```
Comment (haris-bravo-klika) [inline src/components/HomeSettings/index.js:140]: rm commented prop

File: src/components/HomeSettings/index.js [hunk 5]
```diff
@@ -107,6 +111,37 @@ const HomeSettings = ({
...
+            className={classNames(styles.closeProfile)}
+            style={{ width: '30px', color: 'white' }}
```
Comment (haris-bravo-klika) [inline src/components/HomeSettings/index.js:142]: Do not use inline styles, instead write css rules in the appropriate class (i.e. `.closeProfile` in '../styles.module.scss').
To change the icon color, use the icon theme property.

**Review by haris-bravo-klika — CHANGES_REQUESTED** [review]

**Comment by fx1dev** [comment]
@haris-bravo-klika All changes has been made and pushed to the feature branch.

**Review by haris-bravo-klika — APPROVED** [review]

**Pull request merged**